### PR TITLE
chore(cleanup): jshint cleanup for spec

### PR DESCRIPTION
**Pull request by cnishina**
- update gulpfile to use .jshintrc
- address jshint errors in specs

### CODE REVIEW

**Comment by cnishina** [comment]
Why are there no circle ci tests?

**Comment by cnishina** [comment]
checked locally, test suite passes.

File: lib/exitCodes.ts [hunk 1]
```diff
@@ -16,37 +16,24 @@ export class ProtractorError extends IError {
...
-  captureStackTrace() {
-    (Error as any).captureStackTrace(this, this.constructor);
-  }
-
   constructor(logger: Logger, message: string, code: number, error?: Error) {
     super(message);
     this.message = message;
     this.code = code;
-    this.captureStackTrace();
+    // this.captureStackTrace();
```
Comment (juliemr) [inline lib/exitCodes.ts]: Delete me

File: spec/localConf.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+var env = require('./environment.js');
```
Comment (juliemr) [inline spec/localConf.js]: I think this was included by mistake?

**Comment by juliemr** [comment]
LGTM after comments are addressed.

File: spec/localConf.js (hunk 2 above)
Comment (cnishina) [inline spec/localConf.js]: `capabilities: env.capabilities`

File: lib/exitCodes.ts (hunk 1 above)
Comment (cnishina) [inline lib/exitCodes.ts]: Deleted.

File: spec/localConf.js (hunk 2 above)
Comment (cnishina) [inline spec/localConf.js]: The entire file has been removed. Not part of this commit.
